### PR TITLE
Create layer2Account.md

### PR DESCRIPTION
**Pull request by ashleewang888**
Newly added doc on layer 2 account.

### CODE REVIEW

File: versioned_docs/version-1.2-rc1(WIP)/layer2Account.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+id: layer2Account
+title: Layer 2 Account in Godwoken
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Godwoken v1 introduced the registry and registry address, both of which can be seen as alias mechanisms for layer 2 accounts. 
+
+A layer 2 account can be referenced by ID or by script ( a hash of a script usually) itself. However, in some environments, it is also required to access an account by alias. For example, there is no such thing as a Godwoken `ID` or `script` in the EVM environment, and so access to an account can only be made through an ETH address.
```
Comment (Flouse) [inline versioned_docs/version-1.2-rc1(WIP)/layer2Account.md:10]:
```suggestion
A layer 2 account can be referenced by ID or by script (a hash of a script usually) itself. However, in some environments, it is also required to access an account by alias. For example, there is no such thing as a Godwoken `ID` or `script` in the EVM environment, and so access to an account can only be made through an ETH address.
```

File: versioned_docs/version-1.2-rc1(WIP)/layer2Account.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+In this case, a registry is an account. The alias mapping relation is stored in the key value storage of the registry account. The registry address code is:
```
Comment (Flouse) [inline versioned_docs/version-1.2-rc1(WIP)/layer2Account.md:12]:
```suggestion
In this case, a registry is an account. The alias mapping relation is stored in the key value storage of the registry account. The registry address is encoded as follows:
```

File: versioned_docs/version-1.2-rc1(WIP)/layer2Account.md (hunk 1 above)
Comment (ShookLyngs) [reply versioned_docs/version-1.2-rc1(WIP)/layer2Account.md:10]: Maybe `key value storage` can be edited to `key-value storage`?

File: versioned_docs/version-1.2-rc1(WIP)/layer2Account.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+id: layer2Account
+title: Layer 2 Account in Godwoken
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Godwoken v1 introduced the registry and registry address, both of which can be seen as alias mechanisms for layer 2 accounts. 
+
+A layer 2 account can be referenced by ID or by script (a hash of a script usually) itself. However, in some environments, it is also required to access an account by alias. For example, there is no such thing as a Godwoken `ID` or `script` in the EVM environment, and so access to an account can only be made through an ETH address.
```
Comment (xying21) [inline versioned_docs/version-1.2-rc1(WIP)/layer2Account.md:10]: A layer 2 account can be referenced by an ID or by a script (usually a hash of a script). However, in some environments, it is also required to access an account by an alias. For example, in the EVM environment, there is no such thing as a Godwoken `ID` or `script`. Therefore, access to an account can only be made through an ETH address.

File: versioned_docs/version-1.2-rc1(WIP)/layer2Account.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+In this case, a registry is an account. The alias mapping relation is stored in the key-value storage of the registry account. The registry address is encoded as follows:
```
Comment (xying21) [inline versioned_docs/version-1.2-rc1(WIP)/layer2Account.md:12]: In this case, a registry is an account. Alias mappings are stored in the key-value storage of the registry account. The registry address is encoded as follows:

File: versioned_docs/version-1.2-rc1(WIP)/layer2Account.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+```
+(registry ID 4 bytes) | (alias address length 4 bytes) | (alias address n bytes)
+
+```
+
+In Godwoken, every layer 2 account is a tuple of ``(ID, L1 script)`` concept. The ``ID``  is an auto-incrementing number, and the ``L1 script`` is the CKB core data structure unique as to each layer 2 account. `script` , i.e. the CKB core data structure is defined as follows:
```
Comment (xying21) [inline versioned_docs/version-1.2-rc1(WIP)/layer2Account.md:19]: The ``ID``  后多了一个空格

File: versioned_docs/version-1.2-rc1(WIP)/layer2Account.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    code_hash:      Byte32,
+    hash_type:      byte,
+    args:           Bytes,
+}
+
+```
+
+This structure indicates the script that will be loaded when validating an L1 transaction. And, Godwoken re-used this structure on layer 2, in which:
+
+- `code_hash - code_hash` indicates the executable binary used for verifying the layer 2 transaction. If a layer 2 account represents an EVM contract,  the `script.code_hash` of that account will point to the binary file that can verify the EVM transaction.
```
Comment (xying21) [inline versioned_docs/version-1.2-rc1(WIP)/layer2Account.md:33]: an EVM contract,  后多了一个空格

**Review by xying21 — CHANGES_REQUESTED** [review]

**Review by Flouse — APPROVED** [review]

**Pull request merged**